spec/rspec/mocks/argument_matchers_spec.rb

```diff
@@ -295,6 +300,34 @@ module Mocks
         end
       end
 
+      describe "array_excluding" do
+        it "matches array with array_excluding different array" do
+          expect(a_double).to receive(:random_call).with(array_excluding(3, 4))
+          a_double.random_call([1, 2])
+        end
+
+        it "fails array_excluding when args aren't array", :reset => true do
+          expect(a_double).to receive(:msg).with(array_excluding(1, 2, 3))
+          expect {
+            a_double.msg(1, 2, 3)
```

Comment:
It's not exactly clear why this expectation would fail. We expect the array to exclude 1,2,3, but this is what we call it with.
To avoid confusion, do you mind making those 3,4 and 1,2 like in the example above?

Reply:
I changed the message of this to be more clear in what was being tested